ruby/ql/src/change-notes/2023-29-05-improper-ldap-auth-query.md

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
The change note checker is failing because the date format on this file needs to be `YYYY-MM-DD` rather than `YYYY-DD-MM`.